internal/util/k8s/parameters.go

```diff
@@ -34,6 +34,9 @@ const (
 	volSnapNameKey        = csiParameterPrefix + "volumesnapshot/name"
 	volSnapNamespaceKey   = csiParameterPrefix + "volumesnapshot/namespace"
 	volSnapContentNameKey = csiParameterPrefix + "volumesnapshotcontent/name"
+
+	// ClusterNameKey cluster Key, set on RBD image.
+	ClusterNameKey = csiParameterPrefix + "cluster/name"
```

Comment:
This is not a Kubernetes parameter, don't make it one. Use a different prefix, maybe like `csi.ceph.com/cluster/name`? Place it in a different package.